packages/@sanity/cli/test/typegen.test.ts

```diff
@@ -70,6 +70,26 @@ describeCliTest('CLI: `sanity typegen`', () => {
     expect(types.toString()).toMatchSnapshot()
   })
 
+  test('sanity typegen generate: generates query type map', async () => {
+    // Write a prettier config to the output folder, with single quotes. The defeault is double quotes.
```

Comment:
Not sure I understand this comment? We're not writing any prettier configs here, are we?

Reply:
flu barf. thanks! :)